lib/instances/HttpService.lua

```diff
@@ -11,4 +12,21 @@ function HttpService.prototype:JSONDecode(input)
 	return json.decode(input)
 end
 
+function HttpService.prototype:GenerateGUID(wrapInCurlyBraces)
+	local argType = typeof(wrapInCurlyBraces)
+	if wrapInCurlyBraces ~= nil and argType ~= "boolean" then
+		error(("Unable to cast %s to bool"):format(argType), 2)
+	end
+
+	--[[
+		`GenerateGUID` allows any value type for `wrapInCurlyBraces`, but it
+		only omits the curly braces when `wrapInCurlyBraces` is set to `false`
+	]]
+	if wrapInCurlyBraces == false then
+		return "04AEBFEA-87FC-480F-A98B-E5E221007A90"
+	else
+		return "{04AEBFEA-87FC-480F-A98B-E5E221007A90}"

```

Comment:
I'll submit a followup change to generate real GUIDs; it's reasonable for a test to expect this function to work correctly.